src/rdkafka.h

```diff
@@ -6457,20 +6614,26 @@ rd_kafka_group_result_partitions(const rd_kafka_group_result_t *groupres);
  * @sa rd_kafka_AdminOptions_new()
  */
 typedef enum rd_kafka_admin_op_t {
-        RD_KAFKA_ADMIN_OP_ANY = 0,          /**< Default value */
-        RD_KAFKA_ADMIN_OP_CREATETOPICS,     /**< CreateTopics */
-        RD_KAFKA_ADMIN_OP_DELETETOPICS,     /**< DeleteTopics */
-        RD_KAFKA_ADMIN_OP_CREATEPARTITIONS, /**< CreatePartitions */
-        RD_KAFKA_ADMIN_OP_ALTERCONFIGS,     /**< AlterConfigs */
-        RD_KAFKA_ADMIN_OP_DESCRIBECONFIGS,  /**< DescribeConfigs */
-        RD_KAFKA_ADMIN_OP_DELETERECORDS,    /**< DeleteRecords */
-        RD_KAFKA_ADMIN_OP_DELETEGROUPS,     /**< DeleteGroups */
+        RD_KAFKA_ADMIN_OP_ANY = 0,                /**< Default value */
+        RD_KAFKA_ADMIN_OP_CREATETOPICS,           /**< CreateTopics */
+        RD_KAFKA_ADMIN_OP_DELETETOPICS,           /**< DeleteTopics */
+        RD_KAFKA_ADMIN_OP_CREATEPARTITIONS,       /**< CreatePartitions */
+        RD_KAFKA_ADMIN_OP_ALTERCONFIGS,           /**< AlterConfigs */
+        RD_KAFKA_ADMIN_OP_DESCRIBECONFIGS,        /**< DescribeConfigs */
+        RD_KAFKA_ADMIN_OP_DELETERECORDS,          /**< DeleteRecords */
+        RD_KAFKA_ADMIN_OP_LISTCONSUMERGROUPS,     /**< ListConsumerGroups */
```

Comment:
Another question: We are adding things to the middle of a publicly exposed enum, will this break ABI compatibility? 

Say for example someone is dynamically linking librdkafka, and using 
`rd_kafka_AdminOptions_new(rk, RD_KAFKA_ADMIN_OP_DELETEGROUPS)`

He/she now replaces the .so file of librdkafka without recompiling code, and since now enum value points to RD_KAFKA_ADMIN_OP_LISTCONSUMERGROUPS, won't it cause failure?

Reply:
Yes, thanks. I'm moving it at the end of the enum.

Reply:
done